strongbox-commons/src/main/java/org/carlspring/strongbox/util/ClassLoaderFactory.java

```diff
@@ -40,7 +40,7 @@ public static URLClassLoader urlClassLoaderFromPaths(ClassLoader parent,
         }
         catch (IOException e)
         {
-            throw Throwables.propagate(e);
+            throw new RuntimeException(e);
```

Comment:
I think that better to use `UndeclaredThrowableException` here and in similar cases